.env_sample

```diff
@@ -25,3 +25,7 @@ S3_STREAM_KEY='ZS6SNTVEW45MWFOZZHB5'
 S3_STREAM_SECRET_KEY='pFancYkzwzT57TJBdu2pAFdbY/KB+WdoX7kYxAuu7Gg'
 S3_STREAM_BUCKET_DEV='metis-logs-dev'
 S3_STREAM_BUCKET_PROD='metis-logs-prod'
+CLOUDINARY_URL=cloudinaryConfig://477212551875196:UfmSGqVM-h4bvt0726rnXF_5SeQ@stackit
+CLOUDINARY_API_KEY=477212551875196
+CLOUDINARY_API_SECRET=UfmSGqVM-h4bvt0726rnXF_5SeQ
+CLOUDINARY_NAME=stackit
```

Comment:
We must avoid real values in the env_sample. Please use generic values

Reply:
we need this testing account to test the profile picture functionality, these variables are not for prod